src/main/java/com/mitchej123/hodgepodge/util/ASMUtil.java

```diff
@@ -0,0 +1,326 @@
+package com.mitchej123.hodgepodge.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+
+import org.objectweb.asm.tree.AbstractInsnNode;
+import org.objectweb.asm.tree.FieldInsnNode;
+import org.objectweb.asm.tree.IincInsnNode;
+import org.objectweb.asm.tree.InsnNode;
+import org.objectweb.asm.tree.IntInsnNode;
+import org.objectweb.asm.tree.InvokeDynamicInsnNode;
+import org.objectweb.asm.tree.LdcInsnNode;
+import org.objectweb.asm.tree.LineNumberNode;
+import org.objectweb.asm.tree.LocalVariableNode;
+import org.objectweb.asm.tree.MethodInsnNode;
+import org.objectweb.asm.tree.MultiANewArrayInsnNode;
+import org.objectweb.asm.tree.TypeInsnNode;
+import org.objectweb.asm.tree.VarInsnNode;
+
+public class ASMUtil {
```

Comment:
is this class even used ? just delete it

Reply:
It was useful for me, so I figured I should leave it in for others. Instructions don't have a toString implementation, so you can't see what instructions in an InsnList do without debugging the transformer. Should I still remove it?

Reply:
yeah remove it